pjlib/src/pj/ioqueue_winnt.c

```diff
@@ -461,7 +485,11 @@ PJ_DEF(pj_status_t) pj_ioqueue_destroy( pj_ioqueue_t *ioqueue )
 #if PJ_HAS_TCP
     unsigned i;
 #endif
-    pj_ioqueue_key_t *key;
+    pj_ioqueue_key_t *key, *next;
+    pj_time_val stop;
+
+    /* Timeout for keys cancelled ops */
+    enum { WAIT_KEY_MS = 5000 };
```

Comment:
Since the value is quite large (5 sec), better define a macro in the beginning of the file, easier for user to find out and adjust (if necessary).